tools/ci_build/github/linux/test_custom_ops_pytorch_export.sh

```diff
@@ -7,3 +7,7 @@ pip3 install --user /build/Release/dist/*.whl
 export PYTHONPATH=/onnxruntime_src/tools:/usr/local/lib/python3.6/site-packages:$PYTHONPATH
 
 python3 -m pytest -v /onnxruntime_src/tools/test/test_custom_ops_pytorch_exporter.py || exit 1
+
+cd /build/Release && python3 -m pytest -v /onnxruntime_src/onnxruntime/test/python/contrib_ops/onnx_test_trilu.py || exit 1
```

Comment:
Does this mean we'll have to add a line here for every contrib op? Also, is this getting exercised in any CI?

Reply:
Thanks @pranavsharma for mentioning this. I fixed the script to run all tests.
This is running daily on this CI: https://aiinfra.visualstudio.com/Lotus/_build?definitionId=862

Also, all our documentations for contrib ops and test can be found at:
https://github.com/microsoft/onnxruntime/blob/da8a0d0567025a70a520feae3e18569e727fda11/docs/ExportPyTorchCustomOps.md